deployment/vagrant-common/bootstrap.sh

```diff
@@ -157,6 +168,7 @@ if [ ! -e "~/.firstboot" ]; then
   # Setup 
   echo "Installing $ROLE software"
   cd $FWROOT
+  source toolset/setup/linux/prerequisites.sh
```

Comment:
This could probably move WAY up; maybe, we could replace the `sudo pip install progressbar` line way up there since it is called out again in this file. Less duplication.